### PR TITLE
[Concept Entry] MySQL: Indexing

### LINKED ISSUE #4678: [Concept Entry] MySQL indexing
### Reason/inspiration (optional)

We would like a new entry on the `Indexing` concept under MySQL. The entry should go in a new file under `docs/content/mysql/concepts/indexing/indexing.md`.

The entry should include:

- An introductory paragraph that describes the concept
- A ‘Syntax’ section that includes the syntax for the concept (if applicable)
- An ‘Example’ section that shows the concept in use

Please refer to the [concept entry template](https://github.com/Codecademy/docs/blob/main/documentation/concept-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Concept Entrty for Existing Topic

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by Munaelekwa**
### Description

Added an indexing.md file under Mysql. Wrote about indexes in Mysql, with examples.

### Issue Solved
Closes #4678 

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by Munaelekwa** [comment]
Hi @mamtawardhani , my PR is ready for review :)

File: content/mysql/concepts/indexing/indexing.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+('Laptop', 'Electronics', 999.99, 50),
+('Smartphone', 'Electronics', 699.99, 100),
+('Desk', 'Furniture', 199.99, 20);
+```
+
+### Querying with Index
+
+```sql
+SELECT * FROM products WHERE category = 'Electronics' AND price < 800;
+```
```
Comment (cigar-galaxy82) [inline content/mysql/concepts/indexing/indexing.md:142]: after this example show the output

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]
Hi, I have suggested a few changes please modify

**Comment by cigar-galaxy82** [comment]
Hi @Munaelekwa 

Thank you for contributing to Codecademy docs. The entry is very well written, I have suggested a few changes please modify.

**Comment by Munaelekwa** [comment]
Hi @cigar-galaxy82 , i have modified the file to reflect the changes you suggested, kindly review, thanks.

**Comment by cigar-galaxy82** [comment]
Hi @Munaelekwa I see you have made the suggested changes but can you please change the heading in the first suggested also

**Comment by Munaelekwa** [comment]
> Hi @Munaelekwa I see you have made the suggested changes but can you please change the heading in the first suggested also

Please can you be more specific on the line of the heading

**Comment by Munaelekwa** [comment]
> Hi @Munaelekwa I see you have made the suggested changes but can you please change the heading in the first suggested also

Hi @cigar-galaxy82 , I'm yet to get clarity on the further changes you requested

**Comment by cigar-galaxy82** [comment]
Hi @Munaelekwa I mean where you mentioned types of indexes change the heading most used indexes since there are more indexes possible that can be used

**Comment by Munaelekwa** [comment]
Hi @cigar-galaxy82 , I have modified the heading and just made it a sentence. I hope it looks better

**Comment by Munaelekwa** [comment]
Hi @cigar-galaxy82 , I have modified the heading and just made it a sentence. I hope it looks better

**Review by cigar-galaxy82 — APPROVED** [review]

**Review by avdhoottt — CHANGES_REQUESTED** [review]
Hey @Munaelekwa, I've reviewed the entry and left some suggestions. Please make them and ping me whenever you are done. Thanks!!

**Comment by Munaelekwa** [comment]
Hi @avdhoottt , I have committed the changes you suggested

**Review by avdhoottt — APPROVED** [review]
LGTM

File: content/mysql/concepts/indexing/indexing.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+('Sofa', 'Furniture', 499.99, 10);
+```
+
+### Querying with Index
+
+```sql
+SELECT * FROM products WHERE category = 'Electronics' AND price < 800;
+```
+
+**Expected Output**
```
Comment (avdhoottt) [inline content/mysql/concepts/indexing/indexing.md:149]:
```suggestion
Expected Output:
```

**Pull request merged**